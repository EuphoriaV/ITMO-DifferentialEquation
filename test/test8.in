2.42
(4.22 - x)
2.1
4.94