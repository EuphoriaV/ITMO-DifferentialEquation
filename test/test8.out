0.0350429